vignettes/Vaccines.Rmd

```diff
@@ -201,10 +200,9 @@ rtssepiparams2 <- rtssepiparams
 rtssepiparams2$rtss_doses <- c(0, 30, 60)
 rtssepiparams2 <- set_rtss_epi(
   rtssepiparams2,
-  start = rtssepievents$timestep[[1]],
-  end = rtssepievents$timestep[[2]],
+  coverages = rtssepievents$coverages,
+  timesteps = rtssepievents$timesteps,
   age = 5 * month,
-  coverage = 0.8,
   min_wait = 0,
```

Comment:
When only set_rtss_epi is used with no seasonal vaccination, the `min_wait` argument is not required. 
Let's revise the error message for this case, will put note in applicable section of code.